lang/LangPrimSource/PyrPrimitive.cpp

```diff
@@ -4264,7 +4264,7 @@ void initMIDIPrimitives();
 	initMIDIPrimitives();
 #endif
 
-#if !defined(_WIN32) && !defined(SC_IPHONE) && !defined(__OpenBSD__) && !defined(__NetBSD__) && !defined(__APPLE__)
+#if defined __linux__
```

Comment:
Are we confident that this change is safe? It's probably better to be conservative and not assume that __linux__ is defined on all platforms that can use LID (i.e. the old way). However I don't know if there are cases in the wild where this change would be a problem. Personally I'd say stick with the verbose way, exclusion rather than inclusion.